third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -498,8 +505,11 @@ int LibusbOverChromeUsb::LibusbGetDeviceDescriptor(
   return LIBUSB_SUCCESS;
 }
 
-uint8_t LibusbOverChromeUsb::LibusbGetBusNumber(libusb_device* /*dev*/) {
-  return kFakeDeviceBusNumber;
+uint8_t LibusbOverChromeUsb::LibusbGetBusNumber(libusb_device* dev) {
+  if (kBusNumbers.find(dev->chrome_usb_device().device) != kBusNumbers.end()) {
```

Comment:
nit: Let's use an iterator to avoid searching in the map twice.

(Not only because it's unnecessary, but also to avoid the possibility of a bug that someone changes one place but not the other.)

Reply:
Done